### PR TITLE
English translation - extension

### PR DESCRIPTION
**Pull request by christpet**
This is good to go!

### CODE REVIEW

File: src/i18n/resources/en/extension.json [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
+      "back": "back",
+      "messages": "Back to messages",
+      "submissions": "Back to contributions"
     },
-    "manage_subscription": "Gérer mes éclaireurs"
+    "manage_subscription": "Manage my guides"
   },
   "date": {
-    "short": "{{date, DD/MM/YYYY}}",
-    "medium": "Le {{date, DD/MM/YYYY}}"
```
Comment (gregoirelacoste) [inline src/i18n/resources/en/extension.json:27]: We must remove the "Le"

**Review by gregoirelacoste — CHANGES_REQUESTED** [review]

File: src/i18n/resources/en/extension.json (hunk 1 above)
Comment (christpet) [reply src/i18n/resources/en/extension.json:27]: Ah good find ;)

File: src/i18n/resources/en/extension.json [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
+    "short": "{{date, MM/DD/YYYY}}",
+    "medium": "Le {{date, MM/DD/YYYY}}"
```
Comment (christpet) [inline src/i18n/resources/en/extension.json:27]:
```suggestion
    "medium": "{{date, MM/DD/YYYY}}"
```

**Comment by lutangar** [comment]
> https://github.com/dis-moi/extension/pull/854#issuecomment-803881548

**Pull request merged**